### DISCUSSION TITLE
Reagrding Github Octernship

### DISCUSSION THREAD

**Discussion by pushker001** [rustdesk/rustdesk#4619] [General]
I am working on the assignment given for the octernship.
The task is to elevate privileges to run ls -la /root/ with Rust and print the result on the Flutter window.
I created a file named main.rs in which I placed the following code, as shown in the figure. Then, I ran the Flutter command. I am stuck here and don't know what to do next

![Screenshot (16)](https://github.com/rustdesk/rustdesk/assets/50774898/58d8e5dd-9ec2-44b1-b099-432203e122ce)

**Comment by rustdesk** [comment]
Ask ChatGPT `How to run a rust function in flutter?`.

**Comment by pushker001** [comment]
When I try to configure the TOML file, I encounter an error. I have attempted to resolve it but have been unsuccessful.
![Screenshot (17)](https://github.com/rustdesk/rustdesk/assets/50774898/9199f161-eb17-47dd-9114-6a28a5ebf824)
